components/dashboard/DashboardAssignments.js

```diff
@@ -23,27 +23,59 @@ class DashboardAssignments extends Component {
     const assignmentFilters = [
       { name: 'Teams', id: 'teams' },
       { name: 'Favorites', id: 'favorites' },
-      { name: 'All', id: 'all' }
+      { name: 'All', id: 'userContributions' }
```

Comment:
@alyssadelaine and @kamicut The 'all' tab's id has been updated to 'userContributions.' If the campaigns in the user object "all campaigns" array can be filtered to exclude favorites and team assignments, then we can update this tab to "Contributions" and add "all" as a tab that collates all three categories. However, the issue at the moment is that it seems that favorites and assignments also live in the user object "all campaigns" array coming from OSMESA, and that the 'all' tab overrides the selection of the sub-categories.